src/features/saksoversikt/sak.slice.ts

```diff
@@ -940,6 +940,10 @@ export default createSlice({
             state.sak = oppdaterKlageISak(state.sak, action.payload);
         });
 
+        builder.addCase(klageActions.bekreftFormkrav.fulfilled, (state, action) => {
+            state.sak = oppdaterKlageISak(state.sak, action.payload);
```

Comment:
Hva var egentlig konsekvensen av at disse 2 manglet?

Reply:
et eksempel var jo at jeg sendte til attestering, ble navigert til saksoversikten, og hadde knapp for å attestere uten at jeg endret saksbehandler 🤔 så frontenden fikk ikke klage-staten oppdatert